### PR TITLE
LIP30: Update due to new on-chain architecture

### PR DESCRIPTION
**Pull request by ricott1**

### CODE REVIEW

File: proposals/lip-0030.md [hunk 1]
```diff
@@ -325,17 +331,17 @@ accountData = {
     },
     "votes": [
       {
-        "delegateAddress": "c87ea0fa872534cd8fad11193960aa0e5579762af9c2bd23402b6d70fc62f297",
+        "delegateAddress": "83298e0a8a473bb8fd4df30b9230c226c6abdc90fb0030a8439c1d7a44499477",
```
Comment (IkerAlus) [inline proposals/lip-0030.md:334]: Why this and the next 2 delegate addresses are 32 bytes long?

File: proposals/lip-0030.md [hunk 2]
```diff
@@ -53,10 +54,11 @@ Accounts are serialized and deserialized according to the `account` schema. The
...
     2. The legacy address, used before the implementation of [LIP 0018](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0018.md). This value is 8 bytes long.
-2. `balance`: The account balance in Beddows.
-3. `keys`: Accounts that issued a Multisignature Transaction store the transaction-signing rules in this property.
-4. <code>[nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce)</code>: The current nonce, representing the total number of transactions sent from the account.
-5. <code>asset</code>: The account asset stores extra information related to transaction assets. In this LIP we specify the asset schema for the Lisk mainchain.
+2. `token`: This property corresponds to the token module and contains the account balance in Beddows.
+3. `sequence`: This property corresponds to the sequence module and contains the current [nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce), representing the total number of transactions sent from the account.
+4. `keys`: Accounts that issued a Multisignature Transaction store the transaction-signing rules in this property.
```
Comment (IkerAlus) [inline proposals/lip-0030.md:59]: I think now the preferred way for transaction types is to write everything lowercase.

**Review by IkerAlus — CHANGES_REQUESTED** [review]
Consider the small comments. Also, in lines 29 and 39 the whole title of the referenced LIP is copied which is not the style in the rest of LIP links in the document, consider removing those titles too.

The rest LGMT.

File: proposals/lip-0030.md (hunk 1 above)
Comment (ricott1) [reply proposals/lip-0030.md:334]: Nice spot. I was considering public keys rather than addresses. Will fix

File: proposals/lip-0030.md [hunk 3]
```diff
@@ -53,10 +54,11 @@ Accounts are serialized and deserialized according to the `account` schema. The
...
-2. `balance`: The account balance in Beddows.
-3. `keys`: Accounts that issued a Multisignature Transaction store the transaction-signing rules in this property.
-4. <code>[nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce)</code>: The current nonce, representing the total number of transactions sent from the account.
-5. <code>asset</code>: The account asset stores extra information related to transaction assets. In this LIP we specify the asset schema for the Lisk mainchain.
+2. `token`: This property corresponds to the token module and contains the account balance in Beddows.
+3. `sequence`: This property corresponds to the sequence module and contains the current [nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce), representing the total number of transactions sent from the account.
+4. `keys`: Accounts that issued a multisignature registration transaction store the transaction-signing rules in this property.
+5. <code>dpos</code>: This module contains the properties relative to the [Lisk Delegated Proof of Stake](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0023.md) protocol.
```
Comment (MaximeGagnebin) [inline proposals/lip-0030.md:60]:
```suggestion
5. `dpos`: This module contains the properties relative to the [Lisk Delegated Proof of Stake](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0023.md) protocol.
```

**Review by MaximeGagnebin — APPROVED** [review]
Left a small format comment. Looks great!

**Review by IkerAlus — APPROVED** [review]
LGTM

File: proposals/lip-0030.md (hunk 3 above)
Comment (ricott1) [reply proposals/lip-0030.md:60]: Just as a side comment, this does not change the rendered text

Comment (MaximeGagnebin) [reply proposals/lip-0030.md:60]: 👍  I just suggested it as the 3 points above use the `...` syntax.

Comment (ricott1) [reply proposals/lip-0030.md:60]: yeah, i think the autoformatter puts the code tag if there is a link (which is now not there anymore)

File: proposals/lip-0030.md [hunk 4]
```diff
@@ -53,10 +54,11 @@ Accounts are serialized and deserialized according to the `account` schema. The
...
-2. `balance`: The account balance in Beddows.
-3. `keys`: Accounts that issued a Multisignature Transaction store the transaction-signing rules in this property.
-4. <code>[nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce)</code>: The current nonce, representing the total number of transactions sent from the account.
-5. <code>asset</code>: The account asset stores extra information related to transaction assets. In this LIP we specify the asset schema for the Lisk mainchain.
+2. `token`: This property corresponds to the token module and contains the account balance in Beddows.
+3. `sequence`: This property corresponds to the sequence module and contains the current [nonce](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0015.md#nonce), representing the total number of transactions sent from the account.
+4. `keys`: Accounts that issued a multisignature registration transaction store the transaction-signing rules in this property.
+5. `dpos`: This module contains the properties relative to the [Lisk Delegated Proof of Stake](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0023.md) protocol.
```
[the remark this answers is not in the document]
Comment (janhack) [reply proposals/lip-0030.md:60]: Change to spelling used in the protocol documentation.

**Review by janhack — APPROVED** [review]
Looks all good.

**Review by shuse2 — APPROVED** [review]

**Review by karmacoma — APPROVED** [review]

**Pull request merged**